src/m365/spo/commands/listitem/listitem-retentionlabel-remove.spec.ts

```diff
@@ -0,0 +1,269 @@
+      if (opts.url === `https://contoso.sharepoint.com/_api/web/lists(guid'${formatting.encodeQueryParameter(listId)}')/items(1)/SetComplianceTag()`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        listId: listId,
+        webUrl: webUrl,
+        listItemId: 1
+      }
+    });
+    assert(loggerLogToStderrSpy.calledWith());
+  });
+
+  it('removes the retentionlabel based on listTitle when prompt confirmed', async () => {
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async () => (
+      { continue: true }
+    ));
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso.sharepoint.com/_api/web/lists/getByTitle('${formatting.encodeQueryParameter(listTitle)}')/items(1)/SetComplianceTag()`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        listTitle: listTitle,
+        webUrl: webUrl,
+        listItemId: 1
+      }
+    });
+    assert(loggerLogToStderrSpy.calledWith());
+  });
+
+  it('removes the retentionlabel based on listUrl', async () => {
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso.sharepoint.com/_api/web/GetList(@a1)/items(@a2)/SetComplianceTag()?@a1='%2F${formatting.encodeQueryParameter(listUrl)}'&@a2='1'`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        confirm: true,
+        listUrl: listUrl,
+        webUrl: 'https://contoso.sharepoint.com',
+        listItemId: 1
+      }
+    });
+    assert(loggerLogToStderrSpy.calledWith());
+  });
+
+  it('removes the retentionlabel based on listUrl when prompt confirmed (debug)', async () => {
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async () => (
+      { continue: true }
+    ));
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso.sharepoint.com/_api/web/GetList(@a1)/items(@a2)/SetComplianceTag()?@a1='%2F${formatting.encodeQueryParameter(listUrl)}'&@a2='1'`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        listUrl: listUrl,
+        webUrl: 'https://contoso.sharepoint.com',
+        listItemId: 1
+      }
+    });
+    assert(loggerLogToStderrSpy.calledWith());
+  });
+
+  it('correctly handles API OData error', async () => {
+    const errorMessage = 'Something went wrong';
+
+    sinon.stub(request, 'post').callsFake(async () => { throw { error: { error: { message: errorMessage } } }; });
+
+    await assert.rejects(command.action(logger, {
+      options: {
+        debug: false,
```

Comment:
`Debug: false` lines are superfluous. Let's remove those.

Reply:
Hi @nicodecleyre, I think you forgot this one comment :)

Reply:
Right.. Thank you for making me aware of that!